src/app.js

```diff
@@ -0,0 +1,20 @@
+const elasticsearch = require('elasticsearch');
+
+async function indexData(action, settings) {
+    const connectionString = settings.connectionString
+    const client = new elasticsearch.Client({node: `${connectionString}`})
+    const res = await client.index(action.params.obj)
+    return res
```

Comment:
If you're not using the result there is no reason to wait or to put it into a variable. You can directly return `client.index...`

This is the same for the other method as well